modules/administration-guide/partials/assembly_readiness-init-containers.adoc

```diff
@@ -0,0 +1,16 @@
+:parent-context-of-readiness-init-containers: {context}
+
+[id="readiness-init-containers_{context}"]
+= Readiness init containers
+
+:context: readiness-init-containers
+
+{prod-short} Operator installs {prod-short} and starts its containers in the correct order. If a node with {prod-short} is restarted and all {prod-short} containers start simultaneously, some containers may fail because some other component they depend on is not ready. To avoid such failures, the readiness init containers queue the containers for {prod-short} components to start in the correct order.
```

Comment:
If a node with {prod-short} is restarted => 
When one {orch-name} node that is running some {prod-short} workload becomes unavailable or restarts, {orch-name} redistributes the workload in random order.